bitbadges/chain.json

```diff
@@ -7,8 +7,8 @@
   "website": "https://bitbadges.io/",
   "pretty_name": "BitBadges",
   "chain_id": "bitbadges-1",
-  "bech32_prefix": "cosmos",
   "daemon_name": "bitbadgeschaind",
+  "bech32_prefix": "bb",
```

Comment:
Can we separate the bech32_prefix change from the other updates (like those pertaining to version) in separate PRs?
I'm not so sure it's appropriate to register a bech32 prefix that is knowingly incorrect, and personally am leaning toward recommending keeping it as 'cosmos' until the chain has actually been updated to use a new prefix.

Reply:
The prefix has been changed to "bb" via a full chain restart as of a couple days ago (10/23 I believe). It wasn't ideal but probably best long term for all parties. 

As for version updates, it was a full chain restart (so new genesis and everything). This is why such details are overwritten rather than new versions.

Reply:
Oh, I didn't realize that it was actually changed; I'm OK with this, then. Thanks for the info!